core/arch/arm/plat-ls/main.c

```diff
@@ -84,7 +84,7 @@ void plat_cpu_reset_late(void)
 
 #if defined(CFG_BOOT_SECONDARY_REQUEST)
 		/* set secondary entry address */
-		write32(__compiler_bswap32(CFG_TEE_LOAD_ADDR),
+		write32(__compiler_bswap32(TEE_TEXT_VA_START),
```

Comment:
The physical address is required here, not the optee private virtual address of its entry point.